src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -168,7 +168,11 @@ For example, `sub-01_mod-T1w_defacemask.nii.gz`.
 If several scans of the same modality are acquired they MUST be indexed with the
 [`run-<index>`](../99-appendices/09-entities.md#run) key-value pair:
 `_run-1`, `_run-2`, `_run-3`, and so on (only nonnegative integers are allowed as
-run labels). When there is only one scan of a given type the run key MAY be
+run labels).
+If however these scans of the same modality contain different labels
+(for example `_acq-`, `_ce-`, `_desc-`) then the run label and indexing are optional,
+unless the labels are the same.
+When there is only one scan of a given type the run key MAY be
```

Comment:
What about:

```Markdown
If several scans with the same acquisition parameters are acquired in the same session,
they MUST be indexed with the [`run-<index>`](../99-appendices/09-entities.md#run) entity:
`_run-1`, `_run-2`, `_run-3`, and so on (only nonnegative integers are allowed as
run labels).
If different entities apply,
such as a different session indicated by [`ses-<label>`](../99-appendices/09-entities.md#ses),
or different acquisition parameters indicated by [`acq-<label>`](../99-appendices/09-entities.md#acq),
then `run` is not needed to distinguish the scans and MAY be omitted.
```

This replaces the somewhat ambiguous concept of "modality" with acquisition parameters; in almost all cases identical acquisition parameters should imply the same modality.

This may also make the sentence "When there is only one scan of a given type the run key MAY be omitted." unnecessary, though I don't think that it harms clarity to keep it in.

Reply:
This wording sounds fine with me. I've gone ahead and made the changes, sorry for the delayed response.